src/main/java/com/photolocos/enterprise/dao/PhotoDAO.java

```diff
@@ -72,4 +72,9 @@ public boolean createEntry(PhotoDTO photoDTO) throws Exception {
     public Map<Integer, PhotoDTO> fetchAll() {
         return photos;
     }
+
+    @Override
+    public void save(PhotoDTO photo) {
+
+    }
```

Comment:
This isn't saving anything.  It's an empty method.
Code review should focus on reducing technical debt of existing code before adding new.